docs/source/getting-started/tabular-data/computing-basic-statistics.ipynb

```diff
@@ -0,0 +1,472 @@
+    "    privacy_loss=dp.loss_of(epsilon=1.0),\n",
+    "    split_evenly_over=5,\n",
+    "    margins={\n",
+    "        (): dp.Margin(public_info= \"lengths\", max_partition_length=estimated_max_partition_len, max_num_partitions=1),\n",
+    "    },\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In these examples we see some differences between how regular Polars and Polars with DP are used:\n",
+    "\n",
+    "### 1. Specifying the Data\n",
+    "\n",
+    "In a regular query, use `df` directly. In a differentially private query, use `context.query()`. \n",
+    "\n",
+    "### 2. Applying the Method\n",
+    "\n",
+    "In a regular query, you'll often be able to apply the function directly, such as `.sum()`. In a differentially private query, you may need to pass additional parameters. For example, to compute the differentially private sum, pass in the bounds as a tuple: `dp.sum((1,10))`. \n",
+    "\n",
+    "### 3. Collecting the Results \n",
+    "\n",
+    "In a regular query, use `collect()`. In a differentially private query, use `release().collect().`. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Sum"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "To demonstrate the `sum` method, calculate the total number of hours worked for all years in the dataset. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Actual Total Hours:  2962104.0\n"
+     ]
+    }
+   ],
+   "source": [
+    "total_hours_actual = df.select(pl.col(\"HWUSUAL\").sum()).collect().item()\n",
+    "print('Actual Total Hours: ', total_hours_actual)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The query for the sum is essentially the same, but we do need to call `fill_null` because the `sum` method requires a non-nullable input. \n",
+    "\n",
+    "The imputed value can be non-zero and depends on the context of your data. \n",
+    "\n",
+    "Do not use private data to calculate imputed values or bounds: This could leak private information, rendering the differential privacy guarantees meaningless. Instead, choose bounds and imputed values based on prior domain knowledge."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "DP Total Hours:  2969353.065315555\n"
+     ]
+    }
+   ],
+   "source": [
+    "total_hours_dp = context.query().select(\n",
+    "    pl.col(\"HWUSUAL\").fill_null(0.).dp.sum((0,80))\n",
+    ").release().collect().item()\n",
+    "\n",
+    "print('DP Total Hours: ', total_hours_dp)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Mean"
```

Comment:
The same comment as in for sums about explicitly stating that the first code snippet is non-private. The same thing for median, quantiles, and multiple quantiles, too.